src/api/messaging.js

```diff
@@ -0,0 +1,73 @@
+import { handleResponse, handleError } from './apiUtils';
+import { session } from '../models/auth';
+
+export default {
+  getRegion() {
+    const query = `https://dev-k8s.treetracker.org/messaging/region`;
+
+    return fetch(query, {
+      method: 'GET',
+      headers: {
+        'content-type': 'application/json',
+        Authorization: session.token,
+      },
+    })
+      .then(handleResponse)
+      .catch(handleError);
+  },
+  postRegion(payload) {
+    const query = `https://dev-k8s.treetracker.org/messaging/region`;
+    const { id, name, description, created_at } = payload;
+
+    return fetch(query, {
+      method: 'POST',
+      headers: {
+        'content-type': 'application/json',
+        Authorization: session.token,
+      },
+      body: JSON.stringify({ id, name, description, created_at }),
+    });
+  },
+  getRegionById(region_id) {
+    const query = `https://dev-k8s.treetracker.org/messaging/region/${region_id}`;
+
+    return fetch(query, {
+      headers: {
+        Authorization: session.token,
+      },
+    }).then((res) => console.log(res));
```

Comment:
Could you clarify why this API method results in a console.log if successful? Does this fetch request not require error handling?